backend/server/api.go

```diff
@@ -9,40 +9,35 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/go-chi/chi/v5"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/theparanoids/ashirt-server/backend"
 	"github.com/theparanoids/ashirt-server/backend/contentstore"
 	"github.com/theparanoids/ashirt-server/backend/database"
 	"github.com/theparanoids/ashirt-server/backend/dtos"
 	"github.com/theparanoids/ashirt-server/backend/logging"
 	"github.com/theparanoids/ashirt-server/backend/server/middleware"
 	"github.com/theparanoids/ashirt-server/backend/services"
-
-	"github.com/gorilla/mux"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-func API(db *database.Connection, contentStore contentstore.Store, logger logging.Logger) http.Handler {
-	r := mux.NewRouter()
-	metricRouter := r.PathPrefix("").Subrouter()
-	metricRouter.Handle("/api/metrics", promhttp.Handler())
-
-	r.Use(middleware.LogRequests(logger))
+func API(r chi.Router, db *database.Connection, contentStore contentstore.Store, logger logging.Logger) {
 	r.Use(middleware.AuthenticateAppAndInjectCtx(db))
+	r.Use(middleware.LogRequests(logger))
 
 	bindAPIRoutes(r, db, contentStore)
-	return r
+	r.Mount("/metrics", promhttp.Handler())
```

Comment:
I know we're talking about getting rid of metrics, so this may be moot, but does this use the logRequests and Authenticate middlewares? Ideally we'd want this endpoint to not have those.

Reply:
fixed!